src/test/java/com/epam/prejap/teatrees/block/BrickActionTest.java

```diff
@@ -0,0 +1,58 @@
+package com.epam.prejap.teatrees.block;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.Random;
+
+public class BrickActionTest {
+  @DataProvider
+  private static Object[][] brickActionDataProvider() {
+    return new Object[][]{
+      { new OBlock() }
+    };
+  }
+
+  @Test(
+    groups = "blocks",
+    dataProvider = "brickActionDataProvider")
+  public void brickActionHappyPath(Block block) {
+    // Given dp
```

Comment:
```suggestion
    // Given
```

mentionig of dp int this comment in unnecessary. her and onwards